src/static/js/timeslider.js

```diff
@@ -28,11 +30,81 @@ const Cookies = require('./pad_utils').Cookies;
 const randomString = require('./pad_utils').randomString;
 const hooks = require('./pluginfw/hooks');
 
-let token, padId, export_links;
+let token, padId, exportLinks, socket, changesetLoader, BroadcastSlider;
```

Comment:
If I recall correctly, the `socket` and `BroadcastSlider` vars need to be on the `globalThis` object (`window`) because other `.js` files (and maybe code in the HTML?) reference them. Rather than scope them to this file, use `window.socket` and `window.BroadcastSlider` to ensure that these lint fixes don't change behavior.

I realize that it is gross to do `window.foo`, but cleaning that up is out of scope for a lint fix PR.